scripts/deploy.sh

```diff
@@ -6,6 +6,7 @@ yarn assets
 gzip -S .cgz $(find public/assets -name '*.css')
 gzip -S .jgz $(find public/assets -name '*.js')
 bucket-assets --bucket artsy-force-$DEPLOY_ENV
+heroku config:set NODE_MODULES_CACHE=false --app=force-$DEPLOY_ENV
```

Comment:
Checking to see if setting it here may have any effect. (Will revert after test)